scipy/linalg/flapack.pyf.src

```diff
@@ -542,6 +542,40 @@ interface
 
    end subroutine <prefix>gesv
 
+   subroutine <prefix>gesvx(a,lda,b,ldb,n,nrhs,x,ldx,trans,fact,af,ldaf,piv,r,c,rcond,ferr,berr,equed,work,iwork,info)
+
+   ! Solve A * X = B using LU decomposition
+   ! The expert driver of ?GESV with condition number,
+   ! backward/forward error estimates.
+
+     callstatement (*f2py_func)(a,&lda,b,&ldb,&n,&nrhs,x,&ldx,trans,fact,af,&ldaf,piv,&r,&c,rcond,ferr,berr,equed,&work,&iwork,&info);
+     callprotoargument <ftype>*,int*,<ftype>*,int*,int*,int*,<ftype>*,int*,char*,char*,<ftype>*,int*,int*,<ctypereal>*,<ctypereal>*,<ctypereal>*,<ctypereal>*,<ctypereal>*,char*,<ftype>*,<ftype>*,int*
+
+     <ftype> dimension(n,n),check(shape(a,0)==shape(a,1)),intent(in,out,copy,out=as):: a
+     integer depend(a),intent(hide):: lda = shape(a,0)
+     <ftype> dimension(n,nrhs),intent(in,out,copy,out=bs):: b
+     integer depend(b),intent(hide):: ldb = shape(b,0)
+     integer depend(a),intent(hide):: n = shape(a,0)
+     integer depend(b),intent(hide):: nrhs = shape(b,1)
+     <ftype> dimension(n,nrhs),depend(n,nrhs),intent(out):: x
+     integer depend(n),intent(hide):: ldx = n
+     character intent(in),check(*trans=='N'||*trans=='T'||*trans=='C'):: trans
```

Comment:
I'm not sure this is correct --- how exactly f2py works is a bit of a mystery, so it might very well not be the culprit --- but still.
`gemm` does it differently at least (at *that* was written by the original author of f2py)
https://github.com/scipy/scipy/blob/master/scipy/linalg/fblas_l3.pyf.src#L23